buttplug-device-config.json

```diff
@@ -236,7 +237,8 @@
       "btle": {
         "names": [
           "Onyx2.1",
-          "Titan1.1"
+          "Titan1.1",
+          "Oynx+"
```

Comment:
How is this in the JSON but not the YAML!?

Reply:
Damn. I forget to rebuild the JSON during my rebase.